app/models/place.rb

```diff
@@ -79,14 +41,14 @@ def data_set
   # Convert mongoid's geo_near_distance attribute to a Distance object
   # so that we can easily convert it to other units.
```

Comment:
Suggest we remove this comment as it's mongo specific and (I think) no longer relevant

```suggestion
```

Reply:
Done.